ansible_collections/arista/avd/examples/single-dc-l3ls/README.md

```diff
@@ -0,0 +1,734 @@
+            name: VRF1_VLAN12
+            enabled: true
+            ip_address_virtual: 10.10.12.1/24
+```
+
+This defines `VRF1`, with a L3VNI of `1` and two VLANs (`VLAN11` and `VLAN12`).
+Each VLAN has a name and a virtual IP address assigned, which will be used as the default gateway for that particular VLAN.
+
+At the very bottom we define two layer2-only VLANs (`VLAN3401` and `VLAN3402`), these VLANs are only bridged across the fabric, but never routed anywhere:
+
+```yml
+    l2vlans:
+      "3401":
+        name: L2_VLAN3401
+      "3402":
+        name: L2_VLAN3402
+```
+
+## Specifying endpoint connectivity in the EVPN/VXLAN Fabric
+
+As we saw in the previous section, we've now defined all VRFs and VLANs across the fabric. Let's take a closer look at the `ansible-avd-examples/single-dc-l3ls/group_vars/CONNECTED_ENDPOINTS.yml` file, which specifies the connectivity for all endpoints in the fabric (typically servers):
+
+```yml
+---
+servers:
+  dc1-leaf1-server1:
+    adapters:
+    - type: server
+      server_ports: [ PCI1, PCI2 ]
+      switch_ports: [ Ethernet5, Ethernet5 ]
+      switches: [ dc1-leaf1a, dc1-leaf1b ]
+      vlans: 11-12,21-22
+      native_vlan: 4092
+      mode: trunk
+      spanning_tree_portfast: edge
+      port_channel:
+        description: PortChannel dc1-leaf1-server1
+        mode: active
+
+    - type: ilo
+      server_ports: [ iLO ]
+      switch_ports: [ Ethernet5 ]
+      switches: [ dc1-leaf1c ]
+      vlans: 11
+      mode: access
+      spanning_tree_portfast: edge
+
+  dc1-leaf2-server1:
+    adapters:
+    - type: server
+      server_ports: [ PCI1, PCI2 ]
+      switch_ports: [ Ethernet5, Ethernet5 ]
+      switches: [ dc1-leaf2a, dc1-leaf2b ]
+      vlans: 11-12,21-22
+      native_vlan: 4092
+      mode: trunk
+      spanning_tree_portfast: edge
+      port_channel:
+        description: PortChannel dc1-leaf2-server1
+        mode: active
+
+    - type: ilo
+      server_ports: [ iLO ]
+      switch_ports: [ Ethernet5 ]
+      switches: [ dc1-leaf2c ]
+      vlans: 11
+      mode: access
+      spanning_tree_portfast: edge
+```
+
+This defines the settings for the relevant switch ports to which the endpoints connect, in this case the two servers `dc1-leaf1-server1` and `dc1-leaf2-server1`.
+
+If we look closer at the definitions for `dc1-leaf1-server1`, we see the following:
+
+```yml
+  dc1-leaf1-server1:
+    adapters:
+    - type: server
+      server_ports: [ PCI1, PCI2 ]
+      switch_ports: [ Ethernet5, Ethernet5 ]
+      switches: [ dc1-leaf1a, dc1-leaf1b ]
+      vlans: 11-12,21-22
+      native_vlan: 4092
+      mode: trunk
+      spanning_tree_portfast: edge
+      port_channel:
+        description: PortChannel dc1-leaf1-server1
+        mode: active
+
+    - type: ilo
+      server_ports: [ iLO ]
+      switch_ports: [ Ethernet5 ]
+      switches: [ dc1-leaf1c ]
+      vlans: 11
+      mode: access
+      spanning_tree_portfast: edge
+```
+
+- The relevant `adapters` are defined:
```

Comment:
in this example you describe the settings below the yaml. In previous sections you described it above. I feel this style (below) is easier to follow.

Reply:
Updated